TELCODOCS-2005: Telco Day 2 Troubleshooting and maintenance (Intro, General, Cluster) (#85939)
openshift/openshift-docs: modules/telco-troubleshooting-general-describe-pod.adoc

// Module included in the following assemblies:
//
// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-general-troubleshooting.adoc

:_mod-docs-content-type: PROCEDURE
[id="telco-troubleshooting-general-describe-pod_{context}"]
= Describing a pod

Describing a pod gives you information about that pod to help with troubleshooting.

> The `Events` section provides you with detailed information about the pod and the containers inside of that pod.

Reviewer: ```suggestion
The `Events` section provides detailed information about the pod and the containers inside of it.
```

.Procedure

* Describe a pod by running the following command:
+
--
[source,terminal]
----

> $ oc describe po -n <namespace> busybox-1

Reviewer: Should `po` be `pod`?
